public/style/main.css

```diff
@@ -128,6 +164,127 @@ nav div.section {
   overflow-y       : hidden;
   overflow-x       : hidden;
 }
+
+/* BOTTOM-UP CONTROL PANEL MODULES */
```

Comment:
these seem to be styles sitting on top of my styles and the boostrap ones - we need to reevaluate how these are implemented and not override totally contrary classes but instead build these correctly